### DISCUSSION TITLE
Extractor hood

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [Mariusthvdb/custom-icons#7] [Ideas]
something that has always amazed is we don't have an icon for an extractor hood. My [request for that at Mdi](https://github.com/Templarian/MaterialDesign/issues/5738) was never followed up on, so Ill repeat it here ;-)

examples:

<img width="170" alt="103478023-56caeb80-4dc4-11eb-93c4-c06c820192fe" src="https://user-images.githubusercontent.com/33354141/168164042-7e1ab628-a0e5-43f3-8640-e2e00b3406e3.png">
<img width="195" alt="103478026-57fc1880-4dc4-11eb-83c1-833410aacc53" src="https://user-images.githubusercontent.com/33354141/168164048-bba9bcee-058d-40b4-9ae6-d23e847adbd3.png">

**Comment by arallsopp** [comment]
Ok. Taking a go at creating something based around the above.
